Misc/NEWS.d/next/Library/2023-03-21-15-17-07.gh-issue-102871.U9mchn.rst

```diff
@@ -0,0 +1 @@
+Remove support for obsolete browsers from :mod:`webbrowser`
```

Comment:
```suggestion
Remove support for obsolete browsers from :mod:`webbrowser` and add support for the Epiphany (GNOME Web) browser.
```

Reply:
Thanks for the review! I only renamed the class without adding support. Previously, it supported 'Galeon' and 'Epiphany' on Gnome. Galeon is no longer used.